tests/lib/rules/prefer-screen-queries.test.ts

```diff
@@ -6,6 +6,9 @@ const ruleTester = createRuleTester();
 
 ruleTester.run(RULE_NAME, rule, {
   valid: [
+    {
+      code: `const baz = () => 'foo'`
```

Comment:
I don't get this valid case to be honest

Reply:
for the sake of code coverage 🤷  I needed a variable declaration that wasn't the return of a function call (`const foo = baz()`) nor a destructuring assignation (`const { foo } = baz()`)

Reply:
Ah ok, no problem then.